Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM mysql:latest
+MAINTAINER random-user@email.com
+
+ENV MYSQL_USER myuserr
```

Comment:
🛑 **gitleaks** has detected a secret with rule-id `` in commit 5a24453c11599313c69d5ffe98d3b31c25a3a594
If this secret is a true positive, please rotate the secret ASAP and rebase the commit containing the secret.